setup.py

```diff
@@ -58,15 +58,15 @@ def get_version(version_file):
     },
     # Author details
     author="Cybersecurity and Infrastructure Security Agency",
```

Comment:
You missed this:
```suggestion
    author="Mark Feldhousen",
```

Reply:
Doh!  I clicked the commit suggestion button too furiously.  Now there is a craptacular commit message in there.  I'm going to fix it up.  Standby for rebase.

Reply:
Resolved: 
- https://github.com/cisagov/cyhy-logging/pull/1/commits/44c674985e89005a03e92ca6feb82b15a03eb804